pg_exclude.yml

```diff
@@ -1,5 +1,8 @@
 exclude:
   nightly:
     ol/7: [15]
+    el/8: [10]
```

Comment:
Why do we still need exclusion rules for PG10?

Reply:
Kept it in case of previous minors but not needed as you suggested